src/tsserver/nodeServer.ts

```diff
@@ -256,7 +256,7 @@ namespace ts.server {
         /* eslint-enable no-restricted-globals */
 
         if (typeof global !== "undefined" && global.gc) {
-            sys.gc = () => global.gc!();
+            sys.gc = () => global.gc?.();

```

Comment:
This is to address a frustrating lint error due to some combination of `@types/node`, TypeScript, and eslint. Without the postfix `!` I'll get a lint error in CI, with the postfix `!` I get a lint error locally. Using `?.()` seems to satisfy both, and we call this infrequently so the overhead of the null check is negligible.